server/controllers/getTypes.js

```diff
@@ -0,0 +1,25 @@
+const Airtable = require('airtable');
+
+const { Airtable_API_KEY } = process.env;
+if (!process.env.Airtable_API_KEY) {
+  throw new Error('Missing Airtable API KEY env var');
+}
+
+const base = new Airtable({ apiKey: Airtable_API_KEY }).base('appAZnpLnWP0wjAc6');
+exports.getTypes = (req, res) => {
+  const itemType = [];
+  base('Type').select({
+    view: 'Grid view',
+  }).eachPage((records, fetchNextPage) => {
+    // This function (`page`) will get called for each page of records.
+    records.forEach((record) => {
+      itemType.push({ itemType: record._rawJson.fields.Name, id: record._rawJson.fields.Id });
```

Comment:
I would use record.get('Name') and record.get('Id') method instead of going through all the objects in this way